doc/manual/arrays.rst

```diff
@@ -474,56 +474,10 @@ operators ``/`` and ``\`` operate elementwise when the denominator is a scalar.
 Note that comparisons such as ``==`` operate on whole arrays, giving a single
 boolean answer. Use dot operators for elementwise comparisons.
 
-The following built-in functions are also vectorized, whereby the functions act
-elementwise::
-
-    abs abs2 angle cbrt
-    airy airyai airyaiprime airybi airybiprime airyprime
-    acos acosh asin asinh atan atan2 atanh
-    acsc acsch asec asech acot acoth
-    cos  cospi cosh  sin  sinpi sinh  tan  tanh  sinc  cosc
-    csc  csch  sec  sech  cot  coth
-    acosd asind atand asecd acscd acotd
-    cosd  sind  tand  secd  cscd  cotd
-    besselh besseli besselj besselj0 besselj1 besselk bessely bessely0 bessely1
-    exp  erf  erfc  erfinv erfcinv exp2  expm1
-    beta dawson digamma erfcx erfi
-    exponent eta zeta gamma
-    hankelh1 hankelh2
-     ceil  floor  round  trunc
-    isfinite isinf isnan
-    lbeta lfact lgamma
-    log log10 log1p log2
-    copysign max min significand
-    sqrt hypot
-
-Note that there is a difference between :func:`min` and :func:`max`, which operate
-elementwise over multiple array arguments, and :func:`minimum` and :func:`maximum`, which
-find the smallest and largest values within an array.
```

Comment:
No, I don't think specific information on `max` belongs in that section on dot vectorization syntax.  Just leaving it in the same place in `arrays.rst` should be fine for now, without an example.

(The `max` and `maximum` docstrings should cross-reference each other, too, if they aren't already doing so.)